app/views/canonical_pending_transactions/_canonical_pending_transaction.html.erb

```diff
@@ -76,7 +76,7 @@
         <% end %>
       </div>
 
-      <% if !defined?(receipt_upload_button) && !defined?(hide_tags) && organizer_signed_in? && !pt.instance_of?(OpenStruct) && Flipper.enabled?(:transaction_tags_2022_07_29, @event || pt.local_hcb_code.event) %>
+      <% if !@frame && !defined?(hide_tags) && organizer_signed_in? && !pt.instance_of?(OpenStruct) && Flipper.enabled?(:transaction_tags_2022_07_29, @event || pt.local_hcb_code.event) %>
```

Comment:
What's the reason for adding `@frame`?

Reply:
The reason for adding `@frame` was because the add tag button was already hidden on the HcbCode popover, since the menu does not display on top of the modal. If I increase the z-index of `.menu__content` to > 200 though, it seems like it works. Should I do that in this or a separate PR?

Reply:
Cool - could we make it opt-in instead of using `hide_tags`?

Reply:
I'm worried we'll miss a case where we don't want to show tags

Reply:
I'm not sure off the top of my head but that's why I'm worried. Certainly shouldn't be on the receipt bin page.